examples/main/main.cpp

```diff
@@ -681,6 +683,8 @@ int main(int argc, char ** argv) {
         // reset color to default if there is no pending user input
         if (input_echo && (int) embd_inp.size() == n_consumed) {
             console::set_display(console::reset);
+            display = true;
+
```

Comment:
```suggestion
            display = true;
```

Reply:
sorry  I don't understand, you mean remove line 687 ?

Reply:
remove the trailing whitespace.

Reply:
should be fixed